### PR TITLE
util: Add function to run in scope

### PR DESCRIPTION
**Pull request by mrunalp**
This PR adds a function to run a pid under a systemd scope.
This is for supporting running conmon under pod slice.

### CODE REVIEW

**Comment by runcom** [comment]
LGTM

**Comment by cyphar** [comment]
:+1: Though we should probably implement more of the necessary checks to make this work with older systemd versions.

File: utils/utils.go [hunk 1]
```diff
@@ -54,3 +57,37 @@ func Prctl(option int, arg2, arg3, arg4, arg5 uintptr) (err error) {
...
+	var properties []systemdDbus.Property
+	conn, err := systemdDbus.New()
+	if err != nil {
+		return err
+	}
+	properties = append(properties, systemdDbus.PropSlice(slice))
+	properties = append(properties, newProp("PIDs", []uint32{uint32(pid)}))
+	properties = append(properties, newProp("Delegate", true))
+	properties = append(properties, newProp("DefaultDependencies", false))
+	if _, err := conn.StartTransientUnit(unitName, "replace", properties, nil); err != nil && !isUnitExists(err) {
```
Comment (mrunalp) [inline utils/utils.go]: Removed that additional check so we return the err. We could introduce it later if we need it when I finish the integration.

**Comment by runcom** [comment]
Make lint failed @mrunalp

**Comment by mrunalp** [comment]
@runcom Pushed update that should fix it.

**Pull request merged**